darkmoon/api/v1/metadata/views.py

```diff
@@ -0,0 +1,205 @@
+"""This is the router file."""
+
+###########
```

Comment:
What value do these labels have in the codebase?

Reply:
#36 made a ticket to deal with this later this week.